validator/db/kv/migration_source_target_epochs_bucket_test.go

```diff
@@ -14,7 +14,9 @@ import (
 
 func TestStore_migrateSourceTargetEpochsBucketUp(t *testing.T) {
 	numEpochs := uint64(100)
-	numKeys := 50
+	// numKeys should be more than batch size for testing.
```

Comment:
Technically also shouldn't be a multiple of publicKeyMigrationBatchSize

Reply:
Thanks for the spot ! #8523 has been opened to update the test case.